lib/SIL/InstructionUtils.cpp

```diff
@@ -277,6 +277,18 @@ SILValue swift::stripBorrow(SILValue V) {
   return V;
 }
 
+SILValue swift::getSingleNonIncidentalUse(SILValue V) {
```

Comment:
I would expect this to return `SILInstruction *` since that's the type of an operand's user.

Reply:
What about `Operand*` so it lines up with `getSingleUse()`?

Reply:
`Operand*` makes sense